src/data/finsupp.lean

```diff
@@ -402,6 +402,19 @@ if_pos rfl
 @[simp] lemma erase_ne [has_zero β] {a a' : α} {f : α →₀ β} (h : a' ≠ a) : (f.erase a) a' = f a' :=
 if_neg h
 
+lemma erase_single [has_zero β] {a : α} {b : β} : (erase a (single a b)) = 0 := begin
```

Comment:
```suggestion
@[simp] lemma erase_single [has_zero β] {a : α} {b : β} : (erase a (single a b)) = 0 := begin
```
If you tag this lemma with `@[simp]`, then the `simp` tactic will pick it up and start using it.